execute_pipeline.py

```diff
@@ -104,8 +140,22 @@ def _parse_args():
   parser = _CreateArgumentParser()
   flags = parser.parse_args()
 
+  repo_root = flags.reporoot
   pipeline_args = {}
-  repl_vars = {"REPOROOT": flags.reporoot}
+
+  if flags.env:
+    pipeline_id = str(uuid.uuid4())
+    # Use a unique randome temp directory for remote execution.
+    # TODO(ethanbao): Let remote artman decide the temp directory.
+    repo_root = '/tmp/artman/%s' % pipeline_id
+    pipeline_args['pipeline_id'] = pipeline_id
+
+  pipeline_args['repo_root'] = repo_root
+  # TODO(ethanbao): Remove TOOLKIT_HOME var after toolkit_path is removed from
+  # gapic yaml.
+  repl_vars = {"REPOROOT": repo_root,
+               "TOOLKIT_HOME": os.path.join(flags.reporoot, 'toolkit')}
```

Comment:
style nit: use single quotes ('') for normal string -- double quotes for docstring (""" ... """), because it seems that style is common among other files.